CTFd/scoreboard.py

```diff
@@ -5,57 +5,45 @@
 
 scoreboard = Blueprint('scoreboard', __name__)
 
+def get_standings(admin=False, count=-1):
```

Comment:
You should default count to `None` here, not -1 (and also change the `if count == -1` check later to an `if count is not None`)